server/routes/prisons/sessionTemplates/addSessionTemplateController.test.ts

```diff
@@ -290,6 +295,7 @@ describe('Add a session template', () => {
         categoryGroupReferences: Array<string>(),
         hasLocationGroups: 'yes',
         locationGroupReferences: Array<string>(),
+        hideInPublicServices: 'yes',
       }
 
       // When
```

Comment:
What are these comments?

Reply:
The tests were fairly bulky(each test around this code was over 100 lines of code each) - so added these comments to improve clarity (also helped when we duplicated the basic logic between adding the different parts (incentive, locations, category))